src/caseworker-ref-data/components/upload-info-partial-success/upload-info-partial-success.html

```diff
@@ -25,9 +25,19 @@ <h3 class="govuk-heading-m">What went wrong </h3>
           </tbody>
         </table>
         <h3 class="govuk-heading-m">What you need to do</h3>
-        <p class="govuk-body">You’ll need to update the missing information.</p>
-        <p class="govuk-body">Once all the details in the template are correct, you’ll need to <a
-            routerLink="/caseworker-details">upload it again</a>.</p>
+        <ol>
+          <li class="govuk-body">
+            Check you out have the <a routerLink="/caseworker-details">latest version of the staff data upload
```

Comment:
This doesn't read right. "Check you out have..."

I think it's a case of getting rid of "out".

Reply:
@paul-graham Definitely doesn't read right now that I am reading it again